block/pkg/service/service.go

```diff
@@ -44,9 +44,18 @@ type blockService struct {
 func NewBlockService() pb.BlockHandler {
 
 	log.Infof("Init block service finished.\n")
+
+	blkService := "block"
+	backendService := "backend"
+
+	if(os.Getenv("MICRO_ENVIRONMENT") == "k8s"){
```

Comment:
Please use ```k8s``` as constant.
Also check rest of code

Reply:
done.